test/Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests/XmlPersistenceTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests
+{
+    using Microsoft.TestPlatform.Extensions.TrxLogger.XML;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System.IO;
+
+    [TestClass]
+    public class XmlPersistenceTests
```

Comment:
Add test case to ensure this change doesn't impact strings having invalid chars which are escaped properly. 
E.g.: 
1. var str="This string has these \\0 \\v invalid characters"
2. var str =@"This string has these \0 \v invalid characters"